homeassistant/components/conversation/services.yaml

```diff
@@ -14,3 +14,5 @@ process:
       example: homeassistant
       selector:
         conversation_agent:
+
+reload:
```

Comment:
This still requires some more fields like the process service above.
```python
SERVICE_PROCESS_SCHEMA = vol.Schema(
    {
        vol.Required(ATTR_TEXT): cv.string,
        vol.Optional(ATTR_LANGUAGE): cv.string,
        vol.Optional(ATTR_AGENT_ID): agent_id_validator,
    }
)


SERVICE_RELOAD_SCHEMA = vol.Schema(
    {
        vol.Optional(ATTR_LANGUAGE): cv.string,
        vol.Optional(ATTR_AGENT_ID): agent_id_validator,
    }
)
```

Reply:
Thank you, hope it's fine now :) I just copied it from elsewhere and didn't check for the fields.